docs/working_with.rst

```diff
@@ -140,8 +140,8 @@ Namespace instances can be accessed attribute-style or dictionary key-style::
 
 Typical use::
 
-    >>> g = Graph()
-    >>> s = BNode('someone')
+    >>> g = rdflib.Graph()
+    >>> s = rdflib.BNode('someone')
```

Comment:
this would break if you didn't `import rdflib` before, which we don't do anywhere in this file, so it's actually worse than the current situation...

also it's inconsistent to then leave the next line which needs a `from rdflib import RDF` and the `FOAF` from above...